wolfssl/wolfcrypt/settings.h

```diff
@@ -1045,7 +1045,7 @@ extern void uITRON4_free(void *p) ;
                 #define XREALLOC(p, n, h, t) ((void)(h), (void)(t), realloc((p), (n)))
         /* FreeRTOS pvPortRealloc() implementation can be found here:
          * https://github.com/wolfSSL/wolfssl-freertos/pull/3/files */
-        #elif defined(USE_INTEGER_HEAP_MATH) || defined(OPENSSL_EXTRA)
+        #else
```

Comment:
`pvPortRealloc` is not a standard FREERTOS function. We can't force users to support it. Our library will/should fallback to an XMALLOC/XFREE when XREALLOC is not defined. If we have an issue with XREALLOC being expected let's fix that instead please

Reply:
Alright, I will work on updating the check.  The issue is basically that we are using XREALLOC in multiple spots in our code where OPENSSL_EXTRA/USE_INTEGER_HEAP_MATH are not required, so the include guard will need to be updated.
I do not see any fallback to XMALLOC/XFREE for this case currently, in the ticket the customer is running into a compilation error because XREALLOC was not defined when it was expected.

Reply:
You are correct, those two options no longer cover the various cases. Search for internal.c and ssl.c for `#ifndef XREALLOC` and you'll see the pattern.